src/App.js

```diff
@@ -1,8 +1,33 @@
 import React from "react";
 import "./index.css";
 
-const App = () => {
-  return <p>Hello, world!</p>;
+import NavBar from "./Components/NavBar";
+import Contacts from "./Components/Contacts";
+import ContactUserCard from "./Components/ContactUserCard";
+import Post from "./Components/Post";
+import Posts from "./Components/Posts";
+import UserProfile from "./Components/UserProfile";
+
+const App = ()=>{
+  return(
+    <div className="shell">
+      <NavBar/>
+      <UserProfile/>
+      <div className="allPosts">
+        <Posts/>
```

Comment:
Hey Jerry, great job! However, one pretty substantial note: all of your <ContactUserCard />'s and <Post />'s should be written inside of the corresponding components, as opposed to listed beside them in <App />. I'd like you to refactor that and resubmit. If you need clarification, please reach out!